single_epoch.py

```diff
@@ -69,15 +69,15 @@ def epoch_with_dataloader(
 
     val_loop = tqdm(iter(val_loader))
     for i, data in enumerate(val_loop):
-        if i == 5:
+        if i == 50:
             break
         val_loop.set_description(f"Val, epoch: {epoch_id}")
         val_recall, val_precision = test(data.to(device), model, [])
         val_loop.set_postfix_str(f"Recall Val: {val_recall:.4f}")
 
     test_loop = tqdm(iter(test_loader))
     for i, data in enumerate(test_loop):
-        if i == 5:
+        if i == 50:
```

Comment:
We will have to delete these in the final run.

Reply:
should we make this a config parameter? "evaluate_break_at"?